Library/ViewModels/ConfirmDetailsViewModel.swift

```diff
@@ -79,6 +84,12 @@ public class ConfirmDetailsViewModel: ConfirmDetailsViewModelType, ConfirmDetail
     )
     .take(first: 1)
 
+    /// Called when pledge or bonus is updated by backer
+    let additionalPledgeAmount = Signal.merge(
+      self.pledgeAmountDataSignal.map { $0.amount },
+      initialAdditionalPledgeAmount
```

Comment:
Slightly confusing, why is there both an initial `initialAdditionalPledgeAmount` signal and the updated signal `pledgeAmountDataSignal`? Could `initialAdditionalPledgeAmount` be folded into `pledgeAmountDataSignal` instead of doing a merge?